frontend/src/components/serviceCard.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faSoap,
+  faPumpSoap,
+  faWind,
+  faHandSparkles,
+  faHandScissors,
+} from "@fortawesome/free-solid-svg-icons";
+
+import "./../style/card.scss";
+
+const SERVICEICONS = {
+  Shampoo: faSoap,
+  Conditioner: faPumpSoap,
+  Blower: faWind,
+  Manicure: faHandSparkles,
+  Trim: faHandScissors,
+};
+
+const BUTTONSTATES = {
+  pending: ["Start", "Delete"],
+  active: ["Finish", "Stop"],
+  finished: ["", "Resume"],
+  deleted: ["Restore", ""],
+};
+
+class ServiceCard extends Component {
```

Comment:
The name of this component is the same as one of my components, that could be a problem. But don't worry I'll change mine.